packages/video_player/video_player/ios/Classes/messages.m

```diff
@@ -0,0 +1,289 @@
+    result.textureId = nil;
+  }
+  return result;
+}
+- (NSDictionary *)toMap {
+  return
+      [NSDictionary dictionaryWithObjectsAndKeys:(self.textureId ? self.textureId : [NSNull null]),
+                                                 @"textureId", nil];
+}
+@end
+
+@implementation FLTCreateMessage
++ (FLTCreateMessage *)fromMap:(NSDictionary *)dict {
+  FLTCreateMessage *result = [[FLTCreateMessage alloc] init];
+  result.asset = dict[@"asset"];
+  if ((NSNull *)result.asset == [NSNull null]) {
+    result.asset = nil;
+  }
+  result.uri = dict[@"uri"];
+  if ((NSNull *)result.uri == [NSNull null]) {
+    result.uri = nil;
+  }
+  result.packageName = dict[@"packageName"];
+  if ((NSNull *)result.packageName == [NSNull null]) {
+    result.packageName = nil;
+  }
+  result.formatHint = dict[@"formatHint"];
+  if ((NSNull *)result.formatHint == [NSNull null]) {
+    result.formatHint = nil;
+  }
+  return result;
+}
+- (NSDictionary *)toMap {
+  return [NSDictionary
+      dictionaryWithObjectsAndKeys:(self.asset ? self.asset : [NSNull null]), @"asset",
+                                   (self.uri ? self.uri : [NSNull null]), @"uri",
+                                   (self.packageName ? self.packageName : [NSNull null]),
+                                   @"packageName",
+                                   (self.formatHint ? self.formatHint : [NSNull null]),
+                                   @"formatHint", nil];
+}
+@end
+
+@implementation FLTLoopingMessage
++ (FLTLoopingMessage *)fromMap:(NSDictionary *)dict {
+  FLTLoopingMessage *result = [[FLTLoopingMessage alloc] init];
+  result.textureId = dict[@"textureId"];
+  if ((NSNull *)result.textureId == [NSNull null]) {
+    result.textureId = nil;
+  }
+  result.isLooping = dict[@"isLooping"];
+  if ((NSNull *)result.isLooping == [NSNull null]) {
+    result.isLooping = nil;
+  }
+  return result;
+}
+- (NSDictionary *)toMap {
+  return [NSDictionary
+      dictionaryWithObjectsAndKeys:(self.textureId ? self.textureId : [NSNull null]), @"textureId",
+                                   (self.isLooping ? self.isLooping : [NSNull null]), @"isLooping",
+                                   nil];
+}
+@end
+
+@implementation FLTVolumeMessage
++ (FLTVolumeMessage *)fromMap:(NSDictionary *)dict {
+  FLTVolumeMessage *result = [[FLTVolumeMessage alloc] init];
+  result.textureId = dict[@"textureId"];
+  if ((NSNull *)result.textureId == [NSNull null]) {
+    result.textureId = nil;
+  }
+  result.volume = dict[@"volume"];
+  if ((NSNull *)result.volume == [NSNull null]) {
+    result.volume = nil;
+  }
+  return result;
+}
+- (NSDictionary *)toMap {
+  return [NSDictionary
+      dictionaryWithObjectsAndKeys:(self.textureId ? self.textureId : [NSNull null]), @"textureId",
+                                   (self.volume ? self.volume : [NSNull null]), @"volume", nil];
+}
+@end
+
+@implementation FLTPositionMessage
++ (FLTPositionMessage *)fromMap:(NSDictionary *)dict {
```

Comment:
maybe indicate the return object is "nonnull".

Reply:
It's defined in an NS_ASSUME_NONNULL_BEGIN, so it is `nonnull`.

Reply:
Ah, yeah! I always forgot bout the NS_ASSUME_NONNULL thingy.